src/httpd_jsonapi.c

```diff
@@ -3176,6 +3176,72 @@ jsonapi_reply_library_tracks_put_byid(struct httpd_request *hreq)
   return HTTP_OK;
 }
 
+static int
+jsonapi_reply_library_track_playlists(struct httpd_request *hreq)
+{
+  struct query_params query_params;
+  json_object *reply;
+  json_object *items;
+  char *path;
+  const char *track_id;
+  int id;
+  int total;
+  int ret = 0;
+
+  if (!is_modified(hreq->req, DB_ADMIN_DB_MODIFIED))
+    return HTTP_NOTMODIFIED;
+
+  track_id = hreq->uri_parsed->path_parts[3];
+  if (safe_atoi32(track_id, &id) < 0)
```

Comment:
I notice there are a lot of these in the json handlers, perhaps some duplication could be avoided by converting before calling the handler?

Reply:
Thanks for the review, always appreciated. It is difficult to do this outside of the handler, because only the handler currently knows, what parameter and parameter type it expects (we have string, uint32_t and uint64_t in the JSON API and the handler stuff is also used by dacp, rsp, ...). Have to think about that one.